app/js/services/preprocessing.js

```diff
@@ -35,14 +35,37 @@ angular
                 return formatTeamDisplayFilter(teamMap[element.relationships.team.data.id], false);
             }
         },
-				prettifyDate: function(date) {
-					if (date == undefined) { return; };
-					var monthNames = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
-					var year = parseInt(date.substring(0, 4));
-					var month = parseInt(date.substring(5, 7));
-					var day = parseInt(date.substring(8, 10));
-					return monthNames[month - 1] + " " + day + ", "+ year;
-				},
+        changeDate: function(element){
```

Comment:
maybe a better name would be `changeCreatedAndModifiedAttribute`. it's clearer / more specific than `changeDate` even though it's a bit long.

Reply:
what do you think of making this function more versatile as a wrapper function to `convertTimeToEST() `.

e.g. 
```js
convertTimeAttributes: function(element, ['created', 'modified', 'received']) {
  // basically what you have currently element.attributes['created'] = 
  this.convertTimeToEST(element.attributes['created'])
  // etc...
}
```